### ISSUE TITLE
Pain at the anterior aspect of the knee

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10822]
**Preferred term label:**
Pain at the anterior aspect of the knee

**Synonyms**

**Definition (free text, please give PubMed ID)**
Tenderness to palpation at the anterior aspect of the knee

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030839

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Prepatellar bursitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001144
- label: Pain at the anterior aspect of the knee
- parents: [Knee pain (HP:0030839)](https://hpo.jax.org/app/browse/term/HP:0030839)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the front part of the knee, that is, the side of the knee where the patella is located.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10822
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001145
- label: Pain at the posterior aspect of the knee
- parents: [Knee pain (HP:0030839)](https://hpo.jax.org/app/browse/term/HP:0030839)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the back part of the knee, that is, opposite to the side of the knee where the patella is located.
- comment: Pain at the posterior aspect of the knee may be observed with Baker cyst.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10822
- synonyms:
